### ISSUE TITLE
Just Train Network for Pedestrian Detection

### ISSUE THREAD

**Issue by ahkarami** [pjreddie/darknet#747]
Hi,
I have a general question. In the VOC and COCO datasets there are some classes (e.g., 20 or 80 classes (person, car, horse, etc.) for object detection). If one train the network just for one class (_e.g.,_ just the person class) will the accuracy (mAP) of person detection increase?
Another question, does anyone train the YOLO v3 on **pedestrian detection data sets** (_e.g.,_ Caltech)?

**Comment by jjehl** [comment]
one calss =>Not really more accurate but faster. Coco is a good dataset for pedestrian.

**Comment by TheMikeyR** [comment]
@sunshinezhihuo you don't want to detect background so you should not have background included in the class set. 

Just classes=1, and update the filters and files related correctly. 

It's good to have negative images in your dataset (e.g. related images with no persons at all, then darknet will learn to filter the background away upon detection).

**Comment by Paul0629** [comment]
I trained my yolo v3 using coco person and voc person. The loss is below 1. But the detect result is not that good while the pretrained model from the autuor works really well. I don't know why.

**Comment by ahkarami** [comment]
Dear @Paul0629,
I think your work is valuable. Have you used the default hyper-parameters of YOLO v3?
I think, it is better to change the default hyper-parameters for this task. For example change the anchor boxes. You can refer to [training Yolo on the Cityscapes dataset](https://groups.google.com/forum/#!searchin/darknet/pedestrian$20detection|sort:date/darknet/1_HhQwr2BkA/R3P1HPfEAAAJ)

**Comment by Mythili-pixel** [comment]
What should I do for pedestrian detection alone
